src/components/common/Detail.vue

```diff
@@ -50,9 +48,19 @@ export default class Detail extends Vue {
   @Action setDetailComment: ActionBindingIF
   @Action setDetailValidity: ActionBindingIF
 
+  /** Maximum length of comment field in db. */
+  readonly MAX_LENGTH = 4096
+
   private comment: string = null
-  private maxLength: number = 4038
-  private textAreaStyle: string='filled'
+  private textAreaStyle = 'filled'
+
+  private get detailLabel (): string {
+    return `Correction for Incorporation Application. Filed on ${this.getOriginalIAFilingDate}`
+  }
+
+  private get maxLength (): number {
+    return (this.MAX_LENGTH - this.detailLabel.length)
```

Comment:
This is because the label is added to the beginning of the comment text.

Reply:
This conversation is covered by the conversation below, so I'm resolving THIS one.